expression/builtin_string.go

```diff
@@ -3627,3 +3630,164 @@ type loadFileFunctionClass struct {
 func (c *loadFileFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
 	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "load_file")
 }
+
+type weightStringPadding byte
+
+const (
+	// weightStringPaddingNone is used for WEIGHT_STRING(expr) if the expr is non-numeric.
+	weightStringPaddingNone weightStringPadding = 0xFF
+	// weightStringPaddingAsChar is used for WEIGHT_STRING(expr AS CHAR(x)) and the expr is non-numeric.
+	weightStringPaddingAsChar = 0x20
+	// weightStringPaddingAsBinary is used for WEIGHT_STRING(expr as BINARY(x)) and the expr is not null.
+	weightStringPaddingAsBinary = 0x00
+	// weightStringPaddingNull is used for WEIGHT_STRING(expr [AS (CHAR|BINARY)]) for all other cases, it returns null always.
+	weightStringPaddingNull = 0xFE
+)
+
+type weightStringFunctionClass struct {
+	baseFunctionClass
+}
+
+func newWeightStringFunctionClass() *weightStringFunctionClass {
```

Comment:
This function is never used, and other `functionClass` didn't define a `new` function.

Reply:
Yes...it is not used anymore, thanks.